### DISCUSSION TITLE
Doesn't sync titles newer than 12/23

### DISCUSSION THREAD

**Discussion by ghost** [audiamus/BookLibConnect#158] [General]
Hi, I encounter a strange problem when syncing my library: Although I buyed books in '24, they don't show up (correct Account and region). All the books until end of '23 are there (full resync done), but no newer ones. Is this known or is it my fault? 

Thank you

**Comment by audiamus** [comment]
It could still be some odd database synchronization issue. 
You can try to start with a new local database. Rename or delete: 
`C:\Users\<user name>\AppData\Local\audiamus\BookLibConnect\data\audiobooks.db`
BookLibConnect will create a new one.

**Comment by kissfan003** [comment]
I had the same problem with this.  My issue was resolved by deleting the configuration files under C:\Users\<user name>\AppData\Local\audiamus\BookLibConnect\ and redoing the connection in the app.  I think my root cause was 2 factor.  I needed to get the new URL after the 2 factor authentication and it worked.  I was able to get everything from 2024 and earlier.  Thank you Audiamus, you got me started down the correct path to figure it out.
